.scalafmt.conf

```diff
@@ -1,8 +1,7 @@
-version = 3.3.3
+version = 3.4.3
 runner.dialect = scala213
 project.git = true
 docstrings.style = Asterisk
-optIn.blankLineBeforeDocstring = true
```

Comment:
https://scalameta.org/scalafmt/docs/configuration.html#docstringsforceblanklinebefore
default is true